README.md

```diff
@@ -37,19 +49,59 @@ The Vet Clinic database, created with PostgreSQL, stores essential information a
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- GETTING STARTED -->
 
-## 👥 Authors <a name="authors"></a>
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+PostgreSQL installed in your machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  git clone https://github.com/mahdinoori2000/vet_clinic.git
+  cd vet_clinic
+```
+
+
+### Usage
 
-> Mention all of the collaborators of this project.
+To run the project, execute the following command:
+
+```sh
+    psql postgres
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/mahdinoori2000)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mahdinoori23/)
+- GitHub: [mahdinoori2000](https://github.com/mahdinoori2000)
+- LinkedIn: [mahdinoori23](https://www.linkedin.com/in/mahdinoori23/)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Add more tables**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
 
```

Comment:
- [ ] You have done great so far :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
> Find all animals not named Gabumon.

- [ ] According to the project requirement, you were asked to write a query to select animals not named Gabumon, but it seems you did the opposite. Your current query `SELECT * FROM animals WHERE name = 'Gabumon';` returns animals named Gabumon. I'll suggest you rewrite your query correctly and also update the result screenshot. :smile:

Reply:
### What is added newly

- SELECT * FROM animals WHERE name != 'Gabumon';

![amended query](https://github.com/mahdinoori2000/vet_clinic/assets/126486418/4c47f728-475a-40f5-af41-b4189f914a1a)

Reply:
- [ ] Great work writing the query in the terminal but you didn't update this query in the `queries.sql` file. 💪 Kindly update this query. 👍